cypress/e2e/Editor/versions_spec.js

```diff
@@ -41,7 +41,7 @@ describe('Workflow features', () => {
     cy.apiroute('POST', `/graphql-api/graphql`, `converted-article-${ARTICLE_ID}`);
     cy.get('[data-testid=previewVersion]').first().click();
     cy.apiwait(`@converted-article-${ARTICLE_ID}`);
-    cy.get(`article[id=${ARTICLE_ID}]`).should('exist');
+    cy.get(`article`).its('length').should('eq', 3);
```

Comment:
Hvorfor er dette nødvendig?

Reply:
Testen feiler på den gamle måten.  id har blitt fjernet fra article elementet i previewen. Vi har pr på omskrevet versjon av vresion spec så denne er innafor for nå. Den sjekker fortsatt det samme.